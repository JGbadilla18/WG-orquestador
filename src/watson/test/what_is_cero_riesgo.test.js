const testWatson = require('./testWatson');

watsonText = 'Cero Riesgo.'

it(watsonText, () => {
  expect.assertions(1);
  return testWatson(watsonText).then(data => expect(data).toMatchSnapshot());
});
