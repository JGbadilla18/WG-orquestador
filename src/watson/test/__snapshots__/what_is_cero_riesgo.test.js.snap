// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cero Riesgo. 1`] = `
Object {
  "actions": Array [],
  "components": Array [
    Object {
      "buttonType": "web_url",
      "buttons": Array [
        Object {
          "title": "Conoce más aquí",
          "url": "https://blog.corporacionbi.com/tarjeta-credito/blog/adquiere-cero-riesgo-con-tus-tarjetas-de-credito-debito-bi",
        },
      ],
      "response_index": 0,
      "response_type": "button",
      "text": " Cero Riesgo, es un servicio que protege de transacciones no reconocidas en tus Tarjetas de Crédito o Débito Bi, realizando el reintegro de las mismas en 24 horas.
  *  Tiene cobertura para consumos y retiros no reconocidos, por robo y/o extravío.
  *  Tiene un costo de Q10.00 en tarjeta de débito y en tarjeta de crédito el costo varía según la categoría de la misma.",
    },
    Object {
      "response_type": "pause",
      "time": 10000,
      "typing": true,
    },
  ],
}
`;
